guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc

```diff
@@ -4,7 +4,7 @@
 You can use {Project} to configure PXELinux to chainboot iPXE in BIOS mode and boot using the HTTP protocol if you have the following restrictions that prevent you from using PXE:
 
 * A network with unmanaged DHCP servers.
-* A PXE service that is blacklisted on your network or restricted by a firewall.
+* A PXE service that is blocklisted on your network or restricted by a firewall.
```

Comment:
I dislike the allowlist/denylist terminology. Can't we be more generic here and simply say that is somehow unreachable? Because that's the core of the issue.
```suggestion
* An unreachable PXE service, for example by a firewall restriction.
```

Reply:
Works for me.